pandas/tests/reshape/merge/test_merge.py

```diff
@@ -1864,3 +1864,42 @@ def test_merge_index_types(index):
         OrderedDict([('left_data', [1, 2]), ('right_data', [1.0, 2.0])]),
         index=index)
     assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("on,left_on,right_on,left_index,right_index,nms,nm", [
```

Comment:
can you format this so it lines up, its pretty unreadable now

Reply:
Thanks - done